# Created at 2024-09-13T20:32:28.764
org.testng.TestNGException: 
An error occurred while instantiating class Runner_file.RunnerFile. Check to make sure it can be instantiated
	at org.testng.internal.objects.SimpleObjectDispenser.createInstance(SimpleObjectDispenser.java:114)
	at org.testng.internal.objects.SimpleObjectDispenser.dispense(SimpleObjectDispenser.java:41)
	at org.testng.internal.objects.GuiceBasedObjectDispenser.dispenseObject(GuiceBasedObjectDispenser.java:41)
	at org.testng.internal.objects.GuiceBasedObjectDispenser.dispense(GuiceBasedObjectDispenser.java:31)
	at org.testng.internal.ClassImpl.getDefaultInstance(ClassImpl.java:107)
	at org.testng.internal.ClassImpl.getObjects(ClassImpl.java:140)
	at org.testng.internal.IObject.lambda$objects$0(IObject.java:59)
	at java.base/java.util.Optional.map(Optional.java:260)
	at org.testng.internal.IObject.objects(IObject.java:59)
	at org.testng.TestClass.getObjects(TestClass.java:150)
	at org.testng.TestClass.initTestClassesAndInstances(TestClass.java:126)
	at org.testng.TestClass.init(TestClass.java:118)
	at org.testng.TestClass.<init>(TestClass.java:83)
	at org.testng.TestRunner.initMethods(TestRunner.java:473)
	at org.testng.TestRunner.init(TestRunner.java:347)
	at org.testng.TestRunner.init(TestRunner.java:302)
	at org.testng.TestRunner.<init>(TestRunner.java:176)
	at org.testng.SuiteRunner$DefaultTestRunnerFactory.newTestRunner(SuiteRunner.java:636)
	at org.testng.SuiteRunner.<init>(SuiteRunner.java:197)
	at org.testng.TestNG.createSuiteRunner(TestNG.java:1365)
	at org.testng.TestNG.createSuiteRunners(TestNG.java:1339)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1199)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)


